AnkiDroid/src/main/java/com/ichi2/libanki/sched/Sched.java

```diff
@@ -438,7 +438,8 @@ protected Card _getLrnCard(boolean collapse) {
             if (mLrnQueue.getFirst()[0] < cutoff) {
                 long id = mLrnQueue.remove()[1];
                 Card card = mCol.getCard(id);
-                mLrnCount -= card.getLeft() / 1000;
+                /* upstream decrease mLrnCount. We do it in
```

Comment:
Make it obvious to match the two lines, and add a comment on `decrementCount` explaining why
```suggestion
                // mLrnCount -= card.getLeft() / 1000; See decrementCount()
```